libsolidity/ast/Types.h

```diff
@@ -294,6 +294,7 @@ class Type
 		return *m_stackItems;
 	}
 	/// Total number of stack slots occupied by this type. This is the sum of ``sizeOnStack`` of all ``stackItems()``.
+	// TODO: consider the return type to be size_t
```

Comment:
You mean that we should change it? I agree. I think this would also eliminate some conversions to/from `unsigned` because I remember from the signedness warning PR that we're converting it a lot.

```suggestion
	// TODO: consider changing the return type to be size_t
```